profiles/car.lua

```diff
@@ -420,14 +420,16 @@ function process_turn(profile, turn)
       turn.duration = profile.properties.traffic_light_penalty
   end
 
-  if turn.turn_type ~= turn_type.no_turn then
+  -- if turn.turn_type ~= turn_type.no_turn then
+
+  if turn.number_of_roads > 2 then
```

Comment:
If we expose the `is_u_turn` boolean should we expose a flag here `is_a_turn`?

We don't need the number of roads in this function otherwise, no?

Reply:
@TheMarex pointed out that we don't want to make the assumption of `turn.number_of_roads > 2` become equal to `is_a_turn`.

So let's leave this as is for now 😄